scripts/resolve_suggestions_urls.py

```diff
@@ -48,7 +48,7 @@ def remove_suggestions_urls(data) -> int:
     for match in find_suggestion_urls(data):
         parent_pointer = json_path_to_pointer(str(match.context.full_path))
         parent_object = match.context.value
-        del parent_object["suggestions_url"]
+        del parent_object["url"]
```

Comment:
The entire suggestions object should be removed. Currently, this would result in:
```json
"suggestions": {},
```

Reply:
Ok, removing `suggestions` and nested `url` now! Thanks!